src/BenchmarkDotNet/BenchmarkDotNet.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
     <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.61701" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.49" PrivateAssets="contentfiles;analyzers" />
+    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="4.7.1" />
```

Comment:
once the MetadataLoadContext logic is removed from WasmAppBuilder you can get rid of this one too.